scanspec/specs.py

```diff
@@ -2,7 +2,7 @@
 from typing import Any, Callable, Dict, List, Optional, TypeVar
 
 import numpy as np
-from apischema import schema
+from apischema import schema as api_schema
```

Comment:
Why this rename? I'd prefer to keep this as its original name if possible and rename the other thing called `schema`

Reply:
https://github.com/dls-controls/scanspec/issues/6 identified as a resolution to this

Reply:
Added notes to the issue. Upon creating the import lists, there was no longer a conflict between the cli schema command and the imports. As such, I have reverted the naming as suggested.